config.py

```diff
@@ -475,6 +475,33 @@
             '.*[0-9]+.*',
         ],
     ),
+    
+    fi = dict(
+        lang_name = 'Suomi',
+        lang_dir = 'ltr',
+        database = 'fiwiki_p',
+        wikipedia_domain = 'fi.wikipedia.org',
+
+        beginners_link = 'https://fi.wikipedia.org/wiki/Wikipedia:Merkitse_l%C3%A4hteet',
+        beginners_link_title = 'Wikipedia:Merkitse lähteet',
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+
+        citation_needed_category = 'Lähteettömät_artikkelit',
+        citation_needed_templates = EN_CITATION_NEEDED_TEMPLATES + [
+            'Lähde',
```

Comment:
Our parser unfortunately doesn't handle template redirects yet (that's issue #51 and I'll probably solve it as part of issue #62). Are there other spellings for this template, or other templates that redirect to this?

If so, it would be nice to also add a couple of the most common redirects here. Optional, but increases our coverage. The first letter is case-insensitive though, so there's no need to add `lähde`, for example.